pandas/core/groupby/groupby.py

```diff
@@ -2674,7 +2676,11 @@ def _get_cythonized_result(
             Function to be applied to result of Cython function. Should accept
             an array of values as the first argument and type inferences as its
             second argument, i.e. the signature should be
-            (ndarray, Type).
+            (ndarray, Type). Optionally, a third argument can be "masked", to
+            allow for processing specific to nullable values
+        use_nullable_input_arg : bool, default False
+            If True, pass an argument "masked" to the cython function specifying
+            whether or not the input should be treated as masked
```

Comment:
For consistency, with the rest of the args here, would it make sense to call this `needs_nullable`?

Reply:
And group with those args as well.

Reply:
Thanks, your naming suggestions are much clearer